test/router/dnssec/dnssec_suite_test.go

```diff
@@ -39,17 +39,32 @@ func init() {
 	flag.StringVar(&deliveryService, "ds", "changeit", "ds is used to target some dns DS and DNS queries made by traffic router")
 }
 
-var _ = BeforeSuite(func() {
+// var _ = BeforeSuite(func() {
+// 	d = &dnssec.DnssecClient{new(dns.Client)}
+// 	d.Net = "udp"
+
+// 	Expect(nameserver).ToNot(Equal("changeit"), "Pass in a ns flag with the hostname of the traffic router")
+// 	Expect(deliveryService).ToNot(Equal("changeit"), "Pass in a ds flag with the dns label for a DNS delivery service")
+// 	log.Println("Nameserver", nameserver)
+// 	log.Println("DeliveryService", deliveryService)
+// })
+
+// func TestDnssec(t *testing.T) {
+// 	RegisterFailHandler(Fail)
+// 	RunSpecs(t, "Dnssec Suite")
+// }
```

Comment:
Can these commented lines be removed?

Reply:
Maybe. I'm not totally convinced that my new code is doing exactly the same thing as that old code is meant to do. But I am sure it compiles now. Since I'm not sure what the intention was, and I don't feel the need to make a test for something experimental very clean, I'm inclined to leave it in.

Reply:
Sounds good to me